chroma-manager/tests/integration/existing_filesystem_configuration/utils/create_lustre_filesystem.py

```diff
@@ -152,8 +153,7 @@ def create_lustre_filesystem_for_test(self):
                                      'mgt',
                                      self.fsname,
                                      None,
-                                     ['--reformat',
-                                      '--mdt' if self.combined_mgt_mdt else '',
+                                     ['--mdt' if self.combined_mgt_mdt else '',
```

Comment:
Why are we not `--reformat`ting the MGT?

Reply:
Yes.